Sorting/intersect1.js

```diff
@@ -0,0 +1,28 @@
+//this function inputs an array of arrays of numbers, and outputs an array with only the numbers that appear in all the arrays of numbers.
+//it assumes there are no repititions of numbers within arrays, otherwise it breaks.
```

Comment:
**Bonus** can you use a nested dictionary instead of a counter to make this method work even when there is repetition?